chaosazure/__init__.py

```diff
@@ -6,10 +6,16 @@
     discover_probes
 from chaoslib.types import Discovery, DiscoveredActivities, Secrets
 from logzero import logger
+from azure.mgmt.compute import ComputeManagementClient
```

Comment:
We try to sort imports by alphabeptical order:

stdlib modules

third part packages modules

local modules